tools/roundtrip_test.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# Copyright (c) the JPEG XL Project Authors. All rights reserved.
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+# Conformance test tooling test. This is not the JPEG XL conformance test
+# runner. This test that the tooling to generate the conformance test and the
+# conformance test runner work together.
```

Comment:
Let's adapt the doc string from tooling test.

Reply:
Done.